appveyor.yml

```diff
@@ -2,6 +2,38 @@ version: 1.0.{build}
 
 environment:
   matrix:
+    - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2019
+      ARCH: x64-msvc15
```

Comment:
Shouldn't this be `x64-msvc16` to match the Visual Studio version?

Reply:
Agree, it was temporary solution as that was not suppported in [Pthread_WIN32](https://github.com/tango-controls/Pthread_WIN32/)